benchmark/src/jmh/java/com/knuddels/jtokkit/Cl100kParserBenchmark.java

```diff
@@ -0,0 +1,66 @@
+package com.knuddels.jtokkit;
+
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.infra.Blackhole;
+
+import java.util.ArrayList;
+
+public class Cl100kParserBenchmark {
```

Comment:
to be able to have a quick-path that identifies the most common use-cases (i.e. English, ASCII letters, numbers, spaces), we're benchmarking the character classifiers separately